test/end-to-end/stock/stock.setting.js

```diff
@@ -0,0 +1,48 @@
+/* global element, by */
+/* eslint  */
+const helpers = require('../shared/helpers');
+const FU = require('../shared/FormUtils');
+
+const EntryPage = require('./stock.setting.page');
+
+function StockSettingTests() {
+  // the page object
+  const page = new EntryPage();
+
+  // navigate to the page
+  before(() => helpers.navigate('#/stock/setting'));
+
+  it('Should Update Stock Setting', async () => {
+    await page.defineMonthAverageConsumption(5);
+    await page.defaultMinMonthsSecurityStock(4);
+
+    await page.setRadio('yes', 'enable_daily_consumption');
+    await page.setRadio('yes', 'enable_auto_purchase_order_confirmation');
+    await page.setRadio('yes', 'enable_strict_depot_permission');
+    await page.setRadio('yes', 'enable_auto_stock_accounting');
+    await page.setRadio('yes', 'enable_supplier_credit');
+    await element(by.id('algo_msh')).click();
+
```

Comment:
Can we verify that these items have actually changed?  (also in the next 'it').

Reply:
Looks like this was addressed.